app/routes/project/posts/post.js

```diff
@@ -32,6 +32,16 @@ export default Ember.Route.extend({
   },
 
   actions: {
+    save(post) {
+      return post.save().catch((error) => {
+        let payloadContainsValidationErrors = error.errors.some((error) => error.status === 422 );
+
+        if (!payloadContainsValidationErrors) {
+          this.controllerFor('project.posts.post').set('error', error);
```

Comment:
I know this is how we did it originally, but not a fan of what's happening here.